### PR TITLE
Add application code + modified README

### PR DESCRIPTION
**Pull request by tomasmor42**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,11 @@ source name_of_your_virtual_env/bin/activate
...
 export PYTHONPATH="${PYTHONPATH:+/Path/to/this/repo/pytest-aug2019}"
 ```
+* Set the config environment variable:
+
+```
+export APP_SETTINGS=config.py
+```
 * Install required libs in your virtual environment:
 ```
 pip install -r requirements.txt
```
Comment (una-gal) [inline README.md:39]: ERROR: Could not open requirements file: [Errno 2] No such file or directory: 'requirements.txt'

Comment (tomasmor42) [reply README.md:39]: Fixed.

**Review by ddendaas — APPROVED** [review]
Getting a warning, will you tell about this?
..../env/lib/python3.6/site-packages/_pytest/mark/structures.py:324: PytestUnknownMarkWarning: Unknown pytest.mark.slow - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/latest/mark.html
    PytestUnknownMarkWarning,

-- Docs: https://docs.pytest.org/en/latest/warnings.html

**Comment by tomasmor42** [comment]
> Getting a warning, will you tell about this?
> ..../env/lib/python3.6/site-packages/_pytest/mark/structures.py:324: PytestUnknownMarkWarning: Unknown pytest.mark.slow - is this a typo? You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/latest/mark.html
> PytestUnknownMarkWarning,
> 
> -- Docs: https://docs.pytest.org/en/latest/warnings.html

Yes, I will.

**Pull request merged**